src/app/auth/routes.py

```
@@ -147,22 +205,48 @@ def callback() -> Response:
         max_age=settings.cookie.max_age,
         path="/",
     )
+
+    g.current_user = CurrentUser(str(user_id), str(username))
+    g.is_authenticated = True
+    g.authenticated_user_id = str(user_id)
+    g.oauth_credentials = {
+        "consumer_key": OAUTH_CONSUMER_KEY,
```

Flake8 (7.3.0)

[error] undefined name 'OAUTH_CONSUMER_KEY'

(F821)